Make the structure of the project

AbdulSalaamazimi opened the pull request:
I have create two main folder. 1 public 2 src in the public I create html file and in another folder create folder for future work also create in src folder two file 1 App.jsx 2 main.jsx and I some files like gitignore, package.json, ....and more

AbdulSalaamazimi committed:
Make the structure of the project

I have create to main folder. 1 public 2 src in the public I create html file and in another folder create folder for future work. I also create in src folder two file 1 App.jsx 2 main.jsx and I some files like gitignore, package.json, ....and more

leosharifi commented:
**Hey @AbdulSalaamazimi**

Thanks for putting the time.  
But this is not the right structure you created.  
I'll give you the right structure in the next comment.

leosharifi commented:
### Here is the Folder structure 

## 📁 Folder Structure
```text
/src
  /components    // Reusable UI components
  /pages         // Route-level pages (Home, ProjectList, etc.)
  /context       // Context Providers (e.g., AuthContext)
  /hooks         // Custom React hooks (e.g., useFetchProjects)
  /utils         // Helper functions (e.g., API calls)
  /assets        // Images, icons, styles, etc.
  App.jsx        // Main component with routes
  main.jsx       // Entry point

/public
  index.html     // HTML template

.gitignore       // Git ignore file
README.md        // Project documentation
package.json     // Project dependencies and scripts
postcss.config.js      // PostCSS config (if using Tailwind)
vite.config.js         // Vite config (or CRA if preferred)
```

wahidahsharifi commented:
@AbdulSalaamazimi may not have enough time. please improve the pull request @leosharifi & then merge it.

this way @AbdulSalaamazimi still becomes a contributor

leosharifi commented:
Hey @wahidahsharifi  You are right and a very good Idea is as you said I'll merge that but need @AbdulSalaamazimi  in this process! - Thanks

AbdulSalaamazimi committed:
Change some Information

I Delete the file that we dont need in the project for now and make some changes in the folder structure

AbdulSalaamazimi committed:
Add .gitkeep files to include empty folders
